src/psyclone/psyir/backend/fortran.py

```diff
@@ -483,15 +483,20 @@ def gen_use(self, symbol, symbol_table):
                 # This variable is not renamed.
                 only_list.append(dsym.name)
 
+        intrinsic_str = ""
+        if symbol.intrinsic:
+            intrinsic_str = "intrinsic "
```

Comment:
Is this missing the comma-space before intrinsic?
<img width="740" alt="image" src="https://github.com/stfc/PSyclone/assets/1449555/a099e9e8-1439-4649-bebd-a702f66b7ca9">

Reply:
Yes - I've fixed this now